index.html

```diff
@@ -6,11 +6,62 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <!-- Add a link to your css file here -->
 </head>
 
 <body>
     <!-- Add your markup here -->
+    <header class="nav">
+        <nav class="nav-bar">
+            <ul>
+                Home
+            </ul>
+            <ul>
+                About us
+            </ul>
+            <ul>
+                Price List
+            </ul>
+            <ul>
+                Contact us
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section class="info">
+            <h1 class="title">
+                THE VEGAN CAKE SHOP
+            </h1>
+            <h2 class="tagline">
+                Here at the Vegan Cake shop we pride ourselves on tasty cakes that use 100% organic and vegan ingredients. We believe cakes do not need to upset the environment or harm any animals in the process. See our price list for all our pre-made delicious cakes or contact us for bespoke requests.
+            </h2>
+            <button class="order-now">
+                Order Now
+            </button>
+        </section>
+        <section class="cake">
+            <figure class="img1">
+                <img src="https://cdn.shopify.com/s/files/1/2725/9456/products/flavourtown-bakery-london-cakes_vegan-rainbow-new-01_2048x2048.jpg?v=1613475666" alt="picture of a vegan rainbow cake" width="25%">
+            </figure>
+            <figure class="img2">
+                <img src="https://www.vibrantplate.com/wp-content/uploads/2018/12/Vegan-raspberry-cake-10.jpg" alt="picture of vegan raspberry cake" width="25%">
+            </figure>
+            <figure class="img3">
+                <img src="https://cupfulofkale.com/wp-content/uploads/2018/10/whole-cake-735x1102.jpg" alt="picture of a vegan elderflower naked cake" width="25%">
+            </figure>
+            <figure class="img4">
+                <img src="https://eluxemagazine.com/wp-content/uploads/2018/04/unicorn_birthday_cake_recipe.jpg" alt="picture of a vegan unicorn cake" width="25%">
+            </figure>
+        </section>
+        </main>
+        <footer class="social">
+            <a href="https://image.shutterstock.com/image-photo/image-260nw-278925056.jpg"></a>
```

Comment:
```suggestion
            <a href="https://www.facebook.com/"><img src="https://image.shutterstock.com/image-photo/image-260nw-278925056.jpg" alt="facebook" /></a>
```

I don't think these links could show anything the way you have them. What do you think?

Reply:
yep thank you